packages/skill-lesson/path-to-purchase/redeem-dialog.tsx

```diff
@@ -74,11 +74,7 @@ const RedeemDialog = ({open = false, couponId, product}: RedeemDialogProps) => {
             )}
             {title ? (
               <div className="text-lg pt-5 font-medium">
-                <Balancer>
-                  Coupon for {title} by{' '}
-                  {process.env.NEXT_PUBLIC_PARTNER_FIRST_NAME}{' '}
-                  {process.env.NEXT_PUBLIC_PARTNER_LAST_NAME}
-                </Balancer>
```

Comment:
this needs to be more intelligent per product

Reply:
I think we could pass these details into `useCoupon` and then that could pass them along to the RedeemDialog that gets rendered.